src/components/ReportActionItem/MoneyRequestReportPreview/MoneyRequestReportPreviewContent.tsx

```diff
@@ -417,13 +397,16 @@ function MoneyRequestReportPreviewContent({
             ? {maxWidth: reportPreviewStyles.transactionPreviewCarouselStyle.width}
             : {};
 
-    const approvedOrSettledIcon = (iouSettled || isApproved) && (
+    const isIconNeeded = useMemo(() => {
```

Comment:
```suggestion
    // We also show icons in case of Approved or Paid report for easier identification at glance
    const isIconNeeded = useMemo(() => {
```